### ISSUE TITLE
In Ubuntu 20.04 - Getting error - cannot import name 'sip' from 'PyQt5' (/usr/lib/python3/dist-packages/PyQt5/__init__.py)

### ISSUE THREAD

**Issue by rohan-paul** [adamerose/pandasgui#41]
Below is the full error after running `jupyter lab` in a .ipynb notebook file in Ubuntu 20.04

```
import pandas as pd
from pandasgui import show
df = pd.DataFrame(([[1, 2, 3], [4, 5, 6], [7, 8, 9]]), columns=['a', 'b', 'c'])
show(df)
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-dddd71cba70e> in <module>
      1 import pandas as pd
----> 2 from pandasgui import show
      3 from PyQt5 import QtCore, QtGui, QtWidgets, QtWebEngineWidgets
      4 app = QtWidgets.QApplication([])
      5 df = pd.DataFrame(([[1, 2, 3], [4, 5, 6], [7, 8, 9]]), columns=['a', 'b', 'c'])

~/.local/lib/python3.8/site-packages/pandasgui/__init__.py in <module>
----> 1 from pandasgui.gui import show
      2 
      3 __all__ = ["show"]

~/.local/lib/python3.8/site-packages/pandasgui/gui.py in <module>
     10 from PyQt5.QtCore import Qt
     11 
---> 12 from pandasgui.store import Store, PandasGuiDataFrame
     13 from pandasgui.utility import fix_ipython, fix_pyqt, get_logger, as_dict, delete_datasets
     14 from pandasgui.widgets.dataframe_explorer import DataFrameExplorer

~/.local/lib/python3.8/site-packages/pandasgui/store.py in <module>
      3 import pandas as pd
      4 from pandas import DataFrame
----> 5 from PyQt5 import QtCore, QtGui, QtWidgets, sip
      6 from PyQt5.QtCore import Qt
      7 import traceback

ImportError: cannot import name 'sip' from 'PyQt5' (/usr/lib/python3/dist-packages/PyQt5/__init__.py)
```

**Comment by adamerose** [comment]
Weird, looks like you don't have `PyQt5-sip` installed even though it says it's a dependency of `PyQt5` which you do have. Maybe I'll explicitly add it to `setup.py`.

I noticed you posted a 2nd issue with a different error, does that mean you fixed this one?

**Comment by rohan-paul** [comment]
So I finally resolved in Ubuntu 20.04 it as below.

1. Main reason for my case was, I had broken `pyqt5` - so I was not able to uninstall it and also NOT able to install a fresh version of `PyQt5`.

2. So finally had to manually delete  `PyQt5` + `PyQt5-sip` + `PyQtWebEngine`  respective installed directories from the below three locations.

```
/usr/lib/python3/dist-packages

/usr/local/lib/python3.8/dist-packages

/home/user_name/.local/lib/python3.8/site-packages
```

3. Then install them back with `pip install PyQt5 PyQtWebEngine PyQt5-sip`

4. Now, check the installed versions of those packages by running `pip freeze | grep -i pyqt`

Now got the upgraded and correct package version as below

```
PyQt5==5.15.1
PyQt5-sip==12.8.1
PyQtWebEngine==5.15.1
```

5. Then also had to uninstall `pandasgui` by running `pip uninstall pandasgui` and then reinstall it again.

And now `pandasgui`	is running fine.
